src/App.js

```diff
@@ -1,18 +1,20 @@
 import "./App.css";
 import MockAPI from "./mockman/MockAPI";
 import React from 'react';
-import { Route, Routes } from "react-router-dom";
-import Navbar from "./components/Navibar/Navbar";
-import HomePage from "./pages/LandingPage/HomePage/HomePage";
+import { AsideBar } from "./components/Asidebar/Asidebar";
+import { Hero } from "./components/HeroSection/Hero";
+import {LandingPage} from "./pages/LandingPage/LandingPage";
+import { Footer } from "./components/Footer/Footer";
+import { CardHorizontal } from "./components/Card/CardHorizontal";
+
```

Comment:
Suggestion: you can import all files in separate index.js in their folder and use it from there.